vercors/build.sbt

```diff
@@ -35,7 +34,7 @@ lazy val vercors = (project in file("."))
 
     assembly / mainClass := Some("vct.main.Main"),    // Define JAR's entry point
     assemblyMergeStrategy in assembly := {
-      case "logback.xml" => MergeStrategy.first
+      case "logback-test.xml" => MergeStrategy.first

```

Comment:
I don't understand the "logback-test.xml" case, why is it there / what is it for? Can we put this in a comment?

Reply:
For documentation's sake, the file `logback-test.xml` does exist. It is there as a hack to override the logging configuration in `viper`, so it goes through `hre`. The merge strategy shouldn't have said `logback-test.xml`, as it is there to consolidate the logging configuration of viper, which is just called `logback.xml`.